modules/controller/world_model_tracking.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+
+from .. import object_in_world
+
+
+class WorldModelTracking:
+    """
+    keeps track of the real world location of landing pads that
+    are to be visited or have already been vsisited
+    """
+
+    def __init__(self):
+        self.__unconfirmed_positives = []
+        self.__false_positives = []
+        self.confirmed_positive = None
+
+    def __similar(self,
+                  detection1: object_in_world.ObjectInWorld,
+                  detection2: object_in_world.ObjectInWorld) \
+                -> bool:
+        """
+        returns whether detection1 and detection2 are close enough
+        to be considered the same landing pad
+        """
+
+        DISTANCE_THRESHOLD = 2
+        distance = (detection2.position_x - detection1.position_x) ** 2 \
+                 + (detection2.position_y - detection1.position_y) ** 2
+        return distance < DISTANCE_THRESHOLD
```

Comment:
`DISTANCE_THRESHOLD` actually represents the distance squared, rename or use the exponent operator (prefer rename to avoid recalculating every time).